react-dnd/react-dnd.d.ts

```diff
@@ -40,7 +40,7 @@ declare module __ReactDnd {
         spec: DragSourceSpec<P>,
         collect: DragSourceCollector,
         options?: DndOptions<P>
-    ): (componentClass: React.ComponentClass<P>) => DndComponentClass<P>;
+    ): (componentClass: React.ComponentClass<P> | React.StatelessComponent<P>) => DndComponentClass<P>;
```

Comment:
Any reason why you didn't add this to `DropTarget`, `DragDropContext`, and `DragLayer`?